lib/options.js

```diff
@@ -204,6 +204,12 @@ module.exports = optionator({
             default: "-1",
             description: "Number of warnings to trigger nonzero exit code"
         },
+        {
+            option: "exit-on-fatal-error",
+            type: "Boolean",
+            default: "false",
+            description: "Trigger exit code 2 on any fatal errors."
+        },
```

Comment:
This will appear under the `Handling warnings` heading when the user types `eslint -h`. If we want it to be under `Miscellaneous:` then we should move this array element to a different position.